build.gradle.kts

```diff
@@ -8,6 +8,13 @@ configurations {
 }
 
 dependencies {
+  // security
   implementation("org.springframework.boot:spring-boot-starter-webflux")
   implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
+
+  // database
+  implementation("org.springframework.boot:spring-boot-starter-data-jpa")
+  implementation("org.springframework.boot:spring-boot-starter-validation")
+  implementation("org.hibernate:hibernate-core:5.4.24.Final")
+  runtimeOnly("org.postgresql:postgresql")
```

Comment:
Do we use an in-memory database for testing on CI?

Reply:
i would prefer to not to use the in memory db at all if we can help it. 

for integration tests on circle we can run the services via docker-compose.

Reply:
Totally agree -- I didn't see the PostgreSQL container spun up on the CircleCI config which is why I wondered

Reply:
ah - yeh we don't have anything to test yet so left that out of this PR. i actually explicitly added an int test that checks the DB is down!